PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutClient.java

```diff
@@ -157,6 +158,11 @@ private void sendPayPalRequest(
                 } else {
                     environment = Environment.LIVE;
                 }
+                String email = payPalRequest.getUserAuthenticationEmail();
+                AuthConfig authConfig = null;
+                if (email != null) {
+                    authConfig = new AuthConfig(email);
```

Comment:
Can we add a unit test for this portion of the logic?

Reply:
As discussed we can disregard this.